crates/cheatnet/src/constants.rs

```diff
@@ -112,16 +113,30 @@ pub fn build_invoke_transaction(
     }
 }
 
-fn load_contract_class(
+fn read_predeployed_contract_file(
     predeployed_contracts: &Utf8PathBuf,
     contract_path: &str,
-) -> ContractClassV0 {
+) -> String {
     let full_contract_path: PathBuf = predeployed_contracts.join(contract_path).into();
-    let raw_contract_class =
-        fs::read_to_string(full_contract_path).expect("Failed to read predeployed contracts");
+    fs::read_to_string(full_contract_path).expect("Failed to read predeployed contracts")
+}
+
+fn load_v0_contract_class(
+    predeployed_contracts: &Utf8PathBuf,
+    contract_path: &str,
+) -> ContractClassV0 {
+    let raw_contract_class = read_predeployed_contract_file(predeployed_contracts, contract_path);
     ContractClassV0::try_from_json_string(&raw_contract_class).unwrap()
 }
 
+fn load_v1_contract_class(
+    predeployed_contracts: &Utf8PathBuf,
+    contract_path: &str,
+) -> ContractClassV1 {
+    let raw_contract_class = read_predeployed_contract_file(predeployed_contracts, contract_path);
+    ContractClassV1::try_from_json_string(&raw_contract_class).unwrap()
+}
```

Comment:
Those two can be combined in one using generics

Reply:
`try_from_json_string` isn't in any trait and those classes aren't in our codebase - perhaps i don't know how, but it might not be doable?